Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM rust:1.62 AS builder
+WORKDIR /tmp/
+COPY Cargo.toml Cargo.lock ./
+COPY src/lib.rs src/lib.rs
```

Comment:
Any change in `lib.rs` will invalidate incremental builds, so I recommend to follow the pattern we have in Enhacned API:

https://github.com/near/near-enhanced-api-server/blob/main/Dockerfile#L5

```Dockerfile
RUN mkdir src && echo 'fn main() {}' > src/main.rs && cargo build --release && rm -r src
```

Adjusting it to lib-crate:

```Dockerfile
RUN mkdir src && touch src/lib.rs && cargo build --release && rm -r src
```

Reply:
See my comment below

Reply:
Updated. Thanks!